adapters/verizonmedia/verizonmedia.go

```diff
@@ -0,0 +1,189 @@
+package verizonmedia
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type VerizonMediaAdapter struct {
+	http *adapters.HTTPAdapter
+	URI  string
+}
+
+func (a *VerizonMediaAdapter) Name() string {
+	return "verizonmedia"
+}
+
+func (a *VerizonMediaAdapter) SkipNoCookies() bool {
+	return false
+}
+
+func (a *VerizonMediaAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+
+	errs := make([]error, 0, len(request.Imp))
+	if len(request.Imp) == 0 {
+		err := &errortypes.BadInput{
+			Message: "No impression in the bid request",
+		}
+		errs = append(errs, err)
+		return nil, errs
+	}
+
+	errors := make([]error, 0, 1)
```

Comment:
Why declare `errs` and then `errors`? I believe we only need one.
```
func (a *OneMobileAdapter) MakeRequests(requestIn *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {

	request := *requestIn
	errs := make([]error, 0, len(request.Imp))
	if len(request.Imp) == 0 {
		err := &errortypes.BadInput{
			Message: "No impression in the bid request",
		}
		errs = append(errs, err)
		return nil, errs
	}

	errors := make([]error, 0, 1)
```

Reply:
Yes, one is enough, it's fixed in the next update.

Reply:
I improved the coverage to around 90%